easybuild/easyconfigs/g/git/git-2.32.0-GCCcore-10.3.0-nodocs.eb

```diff
@@ -0,0 +1,46 @@
+##
+# Author:    Robert Mijakovic <robert.mijakovic@lxp.lu>
+##
+
+easyblock = 'ConfigureMake'
+
+name = 'git'
+version = '2.32.0'
+versionsuffix = '-nodocs'
+
+homepage = 'https://git-scm.com/'
+description = """Git is a free and open source distributed version control system designed
+to handle everything from small to very large projects with speed and efficiency."""
+
+toolchain = {'name': 'GCCcore', 'version': '10.3.0'}
+
+source_urls = ['https://github.com/git/git/archive']
+sources = ['v%(version)s.tar.gz']
+checksums = ['004697482b6e3b0ae9147580c32efd35869426227f1526f8eafa7950c31def94']
+
+builddependencies = [
+    ('binutils', '2.36.1'),
+    ('Autotools', '20210128'),
+]
+
+dependencies = [
+    ('cURL', '7.76.0'),
+    ('expat', '2.2.9'),
+    ('gettext', '0.21'),
+    ('Perl', '5.32.1'),
+]
+
+osdependencies = [OS_PKG_OPENSSL_DEV]
```

Comment:
@robert-mijakovic Please use the new `OpenSSL` wrapper instead, so:

* add `('OpenSSL', '1.1', '', True)` to `dependencies`
* drop the `osdependencies`

I'll look into enhancing the easyconfig test suite to catch this...

(I also overlooked this in my own PR...)

Reply:
Will fix it. I have noticed in the morning that I had such a mistake in one of the other ECs I have in my stack to commit.